src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Please rename the file to `src/components/Dashboard/index.jsx`

Reply:
done